www/docs/mps/index.php

```diff
@@ -4,64 +4,27 @@
 include_once INCLUDESPATH."easyparliament/people.php";
 
 if (get_http_var('msp')) {
-    list_reps('msps', 'MSPs', 'msp_search');
+    $type = 'msps';
+    $template = 'msp_index';
 } elseif (get_http_var('mla')) {
-    list_reps('mlas', 'MLAs', 'mla_search');
+    $type = 'mlas';
+    $template = 'mla_index';
 } elseif (get_http_var('peer')) {
-    list_reps('peers', 'Members of the House of Lords', 'peer_search');
+    $type = 'peers';
+    $template = 'peer_index';
 } else {
-    list_reps('mps', 'MPs', 'mp_search');
+    $type = 'mps';
+    $template = 'mp_index';
 }
+    $template = 'mp_index';
```

Comment:
Ah no, perhaps it should as only mp_index appears to exist; in which case, none of the other settings of the variable should be there :)